styles.css

```diff
@@ -0,0 +1,125 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.nav_bar {
+  display: flex;
```

Comment:
in order to keep your stylesheet less repetitive and DRY, consider creating a class for the `display: flex;` property since it will be coming in handy all throughout this project.

example

```
.flex {
  display: flex;
}
```

You can truly reuse this class in your HTML tags without ever declaring `display: flex;` anymore